gitops_push

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+echo "DOCKER LOGIN"
+echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin;
+echo "DOCKER PUSH TAG"
+echo ${IMAGE_TAG}
+docker push ${IMAGE_TAG}
+
+git config --global user.email "travis@travis-ci.org"
+git config --global user.name "Travis CI"
```

Comment:
Wellicht een optie om deze ook uit de ENV te halen? Travis CI is niet meer gratis voor open source, dus wellicht dat we op korte termijn over moeten naar een andere CI provider, dan is het wellicht wel handig als dit aanpasbaar is.